run.go

```diff
@@ -91,6 +91,13 @@ func (r *JobRunner) Init() error {
 		log.Error(err)
 	}
 
+	// Set world-write perms on tmpDir, so non-root users can create temp outputs.
+	err = os.Chmod(r.tmpDir, 0777)
```

Comment:
For some reason `os.MkdirAll(r.tmpDir, 0777)` doesn't work, and `/tmp` inside the container ends up with `755` permissions anyway. So I'm setting `777` perms afterwards, which appears to work.

Reply:
Presumably this is umask striking again, I suppose

Reply:
That's right, good point.